entrypoint.sh

```diff
@@ -20,9 +20,11 @@
 # runs "roslaunch carma carma_docker.launch" after setting up the environment
 # such that ROS and CARMA are on the user's PATH
 
-if [ -z "$@" ]; then
+if [ -z "$@" ] || [ "$@" = "bash" ]; then
     # If no other command is passed to this script, run bash
-    source ~/.base-image/init-env.sh; exec "bash"
+    source ~/.base-image/init-env.sh;
+    bash
```

Comment:
I feel like there was a reason I used exec, I can't recall now unfortunately. I want to say it had a lot to do with how environment variables are passed between shells/processes. Have you tested running an image based off this locally, ensuring that it works with different commands?

Reply:
I tested this locally. I was unable to get it to work properly with exec. 
It seemed that using exec caused the environment variables to not be passed to the child process forcing you to call source /opt/carma/install/setup.bash each time. 
By just calling bash I did not have this issue.

Reply:
Sounds good, maybe I was reading some documentation backwards then